packages/python-argus/PKGBUILD

```diff
@@ -0,0 +1,48 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=python-argus
+_pkgname=Argus
+__pkgname=argus
+pkgver=7.56ece87
+pkgrel=1
+pkgdesc='All-in-One OSINT Toolkit.'
+arch=('any')
+groups=('blackarch' 'blackarch-misc' 'blackarch-recon')
+url='https://github.com/jasonxtn/Argus'
+license=('GPL')
+depends=('python')
```

Comment:
Could you list all dependencies https://github.com/jasonxtn/Argus/blob/main/requirements.txt